pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -320,129 +320,129 @@
-<!ENTITY pbBubbleContextMenu-copy.label        "Copy Address">
-<!ENTITY pbBubbleContextMenu-edit.label        "Edit Address">
-<!ENTITY pbBubbleContextMenu-open.label        "Open in Address Book">
-<!ENTITY pbBubbleContextMenu-add.label         "Add to Address Book">
+<!ENTITY pbBubbleContextMenu-remove.label      "Remover endereço">
+<!ENTITY pbBubbleContextMenu-copy.label        "Copiar endereço">
+<!ENTITY pbBubbleContextMenu-edit.label        "Editar endereço">
+<!ENTITY pbBubbleContextMenu-open.label        "Abrir no catálogo de endereços">
+<!ENTITY pbBubbleContextMenu-add.label         "Adicionar ao catálogo de endereços">
 
 <!ENTITY pbInsertHTMLToolbarCmd.label "HTML">
-<!ENTITY pbRemoveTextToolbarCmd.label "Remove text styles">
+<!ENTITY pbRemoveTextToolbarCmd.label "Remover estilos do texto">
 
-<!ENTITY pbComposeSidebar.label "Sidebar">
+<!ENTITY pbComposeSidebar.label "Barra lateral">
 <!ENTITY pbComposeSidebar.accesskey "S">
 
 <!ENTITY pbAddAttachmentCmd.keycode "A">
-<!ENTITY pbReplyButton.label       "Reply">
-<!ENTITY pbReplyAllButton.tooltiptext    "Reply to sender of selected message">
-<!ENTITY pbReplyAllButton.label    "Reply All">
-<!ENTITY pbReplyAllButton.tooltiptext    "Reply to all recipients for selected message">
+<!ENTITY pbReplyButton.label       "Responder">
+<!ENTITY pbReplyAllButton.tooltiptext    "Responder ao remetente da mensagem selecionada">
+<!ENTITY pbReplyAllButton.label    "Responder a todos">
+<!ENTITY pbReplyAllButton.tooltiptext    "Responder a todos os destinatários da mensagem selecionada">
 <!ENTITY pbToggleReplyRecipients.accesskey "R">
 <!ENTITY pbToggleReplyRecipients.key "R">
 
-<!ENTITY pbQuoteMessageButton.label       "Quote">
-<!ENTITY pbQuoteMessageButton.tooltiptext "Quote the selected message">
-<!ENTITY pbQuoteConversationButton.label  "Summarize">
-<!ENTITY pbQuoteConversationButton.tooltiptext "Summarize the conversation">
+<!ENTITY pbQuoteMessageButton.label       "Citar">
+<!ENTITY pbQuoteMessageButton.tooltiptext "Citar a mensagem selecionada">
+<!ENTITY pbQuoteConversationButton.label  "Resumir">
+<!ENTITY pbQuoteConversationButton.tooltiptext "Resumir a conversa">
 <!ENTITY pbToggleReplyQuotingMode.accesskey   "u">
 <!ENTITY pbToggleReplyQuotingMode.key   "O">
 <!ENTITY pbFocusIdentityPicker.key "R">
 
 <!-- Insert Link Dialog -->
 
-<!ENTITY pbInsertLink.label "Enter the Internet address (URL) for this link.">
+<!ENTITY pbInsertLink.label "Digite o endereço de Internet (URL) para este link.">
 <!ENTITY pbInsertLink.accesskey "E">
 
-<!ENTITY pbInsertImage.label "Enter a Internet address (URL) or local file location:">
+<!ENTITY pbInsertImage.label "Digite um endereço de Internet (URL) ou o caminho de um arquivo local:">
 <!ENTITY pbInsertImage.accesskey "E">
 
-<!ENTITY insertListButton.tooltip "Apply or remove bulleted or numbered list">
+<!ENTITY insertListButton.tooltip "Aplicar ou remover lista numerada ou com marcadores">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY insertListButton.tooltip "Aplicar ou excluir lista numerada ou com marcadores">